firefox_puppeteer/ui/windows.py

```diff
@@ -424,14 +429,21 @@ def navbar(self):
         See the :class:`~ui.navbar.NavBar` reference.
         """
 
-    @use_class_as_property('ui.tabbar.Tabs')
+    @property
```

Comment:
Please watch exactly! We have to pass-in a separate argument to the constructor, which is not possible via a decorator.